Dockerfile

```diff
@@ -41,3 +42,4 @@ RUN ln -s /usr/include/oracle/12.2/client64 /usr/lib/oracle/12.2/client64/includ
 
 COPY rootfs /
 WORKDIR /var/www/owncloud
+ENTRYPOINT ["entrypoint"]
```

Comment:
Entrypoints should include the full path

Reply:
done